brave/brave-core
# Adds different text to grant finish screen

Resolves https://github.com/brave/brave-browser/issues/4814

UI PR: https://github.com/brave/brave-ui/pull/499

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on

## Test Plan:
Test plan 1:
- enable rewards
- claim grant
- make sure that text on a final screen is ok (ugp text)

Test plan 1:
- enable rewards
- restore wallet with ad grant waiting
- claim ad grant
- make sure that text on a final screen is ok (ads text)



## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
